assets/js/src/core/components/horizontal-scroll/horizontal-scroll.styles.ts

```diff
@@ -0,0 +1,34 @@
+/**
+* Pimcore
+*
+* This source file is available under two different licenses:
+* - Pimcore Open Core License (POCL)
+* - Pimcore Commercial License (PCL)
+* Full copyright and license information is available in
+* LICENSE.md which is distributed with this source code.
+*
+*  @copyright  Copyright (c) Pimcore GmbH (http://www.pimcore.org)
+*  @license    https://github.com/pimcore/studio-ui-bundle/blob/1.x/LICENSE.md POCL and PCL
+*/
+
+import { createStyles } from 'antd-style'
+
+export const useStyles = createStyles(({ css }, { scrollWidth, hideElement }) => {
+  return {
+    scrollContainer: css`
+        visibility: ${hideElement === true ? 'hidden' : 'visible'};
```

Comment:
This line confuses me a bit. Would that not hide the complete element?

Reply:
of course - this was left over I missed. 
I just updated it.